extensions/smallrye-stork/runtime/src/main/java/io/quarkus/stork/SmallRyeStorkRegistrationRecorder.java

```diff
@@ -0,0 +1,74 @@
+package io.quarkus.stork;
```

Comment:
This is not API, it should be in the `io.quarkus.stork.runtime` package.

Reply:
ok, will do in a separated PR